tests_pytest/pytorch_tests/integration_tests/core/quantization/test_set_node_quantization_config.py

```diff
@@ -0,0 +1,209 @@
+        default_weight_attr_config = AttributeQuantizationConfig(
+            weights_quantization_method=QuantizationMethod.POWER_OF_TWO,
+            weights_n_bits=8,
+            weights_per_channel_threshold=False,
+            enable_weights_quantization=False,
+            # TODO: this will changed to True once implementing multi-attributes quantization
+            lut_values_bitwidth=None)
+
+        # define a quantization config to quantize the kernel (for layers where there is a kernel attribute).
+        kernel_base_config = AttributeQuantizationConfig(
+            weights_quantization_method=QuantizationMethod.SYMMETRIC,
+            weights_n_bits=8,
+            weights_per_channel_threshold=True,
+            enable_weights_quantization=True,
+            lut_values_bitwidth=None)
+
+        # define a quantization config to quantize the bias (for layers where there is a bias attribute).
+        bias_config = AttributeQuantizationConfig(
+            weights_quantization_method=QuantizationMethod.POWER_OF_TWO,
+            weights_n_bits=FLOAT_BITWIDTH,
+            weights_per_channel_threshold=False,
+            enable_weights_quantization=False,
+            lut_values_bitwidth=None)
+
+        base_cfg = schema.OpQuantizationConfig(
+            default_weight_attr_config=default_weight_attr_config,
+            attr_weights_configs_mapping={KERNEL_ATTR: kernel_base_config, BIAS_ATTR: bias_config},
+            activation_quantization_method=QuantizationMethod.POWER_OF_TWO,
+            activation_n_bits=8,
+            supported_input_activation_n_bits=8,
+            enable_activation_quantization=True,
+            quantization_preserving=False,
+            fixed_scale=None,
+            fixed_zero_point=None,
+            simd_size=32,
+            signedness=Signedness.AUTO)
+
+        default_config = schema.OpQuantizationConfig(
+            default_weight_attr_config=default_weight_attr_config,
+            attr_weights_configs_mapping={},
+            activation_quantization_method=QuantizationMethod.POWER_OF_TWO,
+            activation_n_bits=8,
+            supported_input_activation_n_bits=8,
+            enable_activation_quantization=True,
+            quantization_preserving=False,
+            fixed_scale=None,
+            fixed_zero_point=None,
+            simd_size=32,
+            signedness=Signedness.AUTO)
+
+        mx_cfg_list = [base_cfg]
+        for n in [2, 4, 16]:
+            mx_cfg_list.append(base_cfg.clone_and_edit(attr_to_edit={KERNEL_ATTR: {WEIGHTS_N_BITS: n}}))
+        mx_cfg_list.append(
+            base_cfg.clone_and_edit(attr_to_edit={KERNEL_ATTR: {WEIGHTS_N_BITS: 4}})
+        )
+
+        return base_cfg, mx_cfg_list, default_config
+
+    def get_tpc(self):
+        base_cfg, mx_cfg_list, default_config = self.get_op_qco()
+
+        tpc = generate_tpc(default_config=default_config, base_config=base_cfg, mixed_precision_cfg_list=mx_cfg_list,
```

Comment:
don't use this function, just construct a TargetPlatformCapabilities object by hand